ReactWindows/ReactNative.Shared/Bridge/Queue/MessageQueueThread.cs

```diff
@@ -202,13 +217,13 @@ public SingleBackgroundMessageQueueThread(string name, Action<Exception> handler
                 _queue = new BlockingCollection<Action>();
                 _indicator = new ThreadLocal<bool>();
                 _doneHandle = new ManualResetEvent(false);
-                _asyncAction = ThreadPool.RunAsync(_ =>
-                {
-                    _indicator.Value = true;
-                    Run();
-                    _doneHandle.Set();
-                },
-                WorkItemPriority.Normal);
+                Task.Run(() =>
+                    {
+                        _indicator.Value = true;
+                        Run();
+                        _doneHandle.Set();
+                    }
+                );
```

Comment:
Looking across the code base, it looks like we only close curly braces on newlines, and close parentheses on-whatever-line.  E.g., this would be:

```
Task.Run(() =>
{
    ...
});
```